src/pages/Accounts/consts.ts

```diff
@@ -0,0 +1,14 @@
+export const assetsItems = [
+  { label: 'cryptos', value: 'cryptos' },
+  { label: 'nfts', value: 'nfts' },
+];
```

Comment:
Iroha doesn't have this separation of entities. Let's not impl these tabs for now.